x-pack/plugins/aiops/server/routes/explain_log_rate_spikes.ts

```diff
@@ -151,6 +156,105 @@ export const defineExplainLogRateSpikesRoute = (
           }
         }
 
+        if (changePoints?.length === 0) {
+          push(
+            updateLoadingStateAction({
+              ccsWarning: false,
+              loaded: 1,
+              loadingState: `Done.`,
```

Comment:
Is this string of text displayed to the user in the UI?

Reply:
Added an item to the meta issue to translate those messages, there are more from previous PRs too. https://github.com/elastic/kibana/issues/136265